<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Legends</class>
 <widget class="QWidget" name="Legends">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>323</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Index</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="publicgroupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>4</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Public Space</string>
   </property>
   <widget class="QLabel" name="siteAreaLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Park</string>
    </property>
   </widget>
   <widget class="QLabel" name="GreenRatioLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
   </widget>
   <widget class="QLabel" name="parkLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="squareLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="buildinggroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Building</string>
   </property>
   <widget class="QLabel" name="GreenRatioLabel_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Office</string>
    </property>
   </widget>
   <widget class="QLabel" name="siteAreaLabel_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Residential</string>
    </property>
   </widget>
   <widget class="QLabel" name="GreenRatioLabel_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cultural</string>
    </property>
   </widget>
   <widget class="QLabel" name="GreenRatioLabel_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Commercial</string>
    </property>
   </widget>
   <widget class="QLabel" name="GreenRatioLabel_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hotel</string>
    </property>
   </widget>
   <widget class="QLabel" name="GreenRatioLabel_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Civic</string>
    </property>
   </widget>
   <widget class="QLabel" name="officeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="residentialLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="culturalLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>89</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="commercialLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="civicColorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="hotelLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>149</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
